src/contract/utils.ts

```diff
@@ -2,8 +2,9 @@ import { xdr, cereal } from "@stellar/stellar-base";
 import type { AssembledTransaction } from "./assembled_transaction";
 
 /**
- * The default timeout for waiting for a transaction to be included in a block.
+ * The default timebounds, in seconds, during which a transaction will be valid.
```

Comment:
cc @chadoh I forget, is this from the creation of the transaction, assembly, signing, or submission? :sweat_smile: we should clarify that here

Reply:
I changed this wording, and i might've been mistaken, because I think I saw it being used on a tx builder's `setTimeout()` method

Reply:
Made an attempt at addressing this in 462e5f6

Reply:
You need it during initial simulation, but it also gets reset to this value again right before signing.

Reply:
To withdrawl ?